app/src/main/java/io/horizontalsystems/bankwallet/core/providers/AppConfigProvider.kt

```diff
@@ -32,7 +32,7 @@ class AppConfigProvider : IAppConfigProvider, ILanguageConfigProvider, IAppConfi
     override val guidesUrl = App.instance.getString(R.string.guidesUrl)
     override val faqUrl = App.instance.getString(R.string.faqUrl)
     override val fiatDecimal: Int = 2
-    override val maxDecimal: Int = 8
+    override val maxDecimal: Int = 9
```

Comment:
Hi @herou. We will work on the logic of decimals by another issue. At the moment, please set `maxDecimal` to 8